lms/static/js/edxnotes/views/search_box.js

```diff
@@ -10,7 +10,7 @@ define([
         },
 
         errorMessage: gettext('An error has occurred. Make sure that you are connected to the Internet, and then try refreshing the page.'),
-        emptyFieldMessage: gettext('Please enter a term in the search field.'),
+        emptyFieldMessage: gettext('Please enter a term in the <a href="#search-notes-input">search field</a>.'),
```

Comment:
@polesye, thanks for the assist. Happy to go with your method. I've tried it out locally and I'm running into an issue where the link is being rendered as text and not as HTML. Here's a screengrab:

![screen shot 2015-01-13 at 10 29 45 am](https://cloud.githubusercontent.com/assets/163763/5723083/47f007a8-9b0f-11e4-9cc4-7f0fc95dae68.png)

Any thoughts?